### ISSUE TITLE
Build-env not found after update

### ISSUE THREAD

**Issue by heyarne** [dokku/dokku#3646]
I just updated to dokku v0.18.1 on CentOS (via `yum update`) and I'm now receiving a warning every time I run `dokku $something`. I tried `yum reinstall dokku` with no success:

```
$ dokku --version
2019/08/13 16:10:38 open /var/lib/dokku/plugins/available/build-env/plugin.toml: no such file or directory
dokku version 0.18.1
$ ls -a /var/lib/dokku/plugins/available/build-env/
. ..
$ ls -a /var/lib/dokku/core-plugins/available/build-env/
. ..
```

Any suggestion how I can fix this?

**Comment by josegonzalez** [comment]
You'll need to remove the `build-env` plugin. I'll fix this in 0.18.2.

**Comment by josegonzalez** [comment]
Closing as I have a pull request open for this.

Also, wow, people actually install via the rpms!
